java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2412,4 +2416,344 @@ void testCreateDurationNanoseconds() {
+
+    ColumnVector res = ColumnVector.fromLists(DType.STRING, mainList, mainList2);
+    HostColumnVector hcv = res.copyToHost();
+    try {
+      List<List<String>> ret = hcv.getList(1);
+      System.out.println("Element =" + ret.get(0));
+      System.out.println("Element =" + ret.get(1));
+      //add asserts
+    } catch (Exception e) {
+      e.printStackTrace();
+    } finally {
+      res.close();
+      hcv.close();
+    }
+  }
+
+  @Test
+  void testListOfListsCvDoubles() {
+    List<Double> list = new ArrayList<>();
+    list.add(1.1);
+    list.add(2.2);
+    list.add(3.3);
+    List<Double> list2 = new ArrayList<>();
+    list2.add(4.4);
+    list2.add(5.5);
+    list2.add(6.6);
+    List<List<Double>> mainList = new ArrayList<>();
+    mainList.add(list);
+    mainList.add(list2);
+    List<Double> list3 = new ArrayList<>();
+    list3.add(10.1);
+    list3.add(20.2);
+    list3.add(30.3);
+    List<List<Double>> mainList2 = new ArrayList<>();
+    mainList2.add(list3);
+
+    ColumnVector res = ColumnVector.fromLists(DType.FLOAT64, mainList, mainList2);
+    HostColumnVector hcv = res.copyToHost();
+    try {
+      List<Double> ret = hcv.getList(1);
+      System.out.println("Element =" + ret.get(0));
+      //add asserts
+    } catch (Exception e) {
+      e.printStackTrace();
+    } finally {
+      res.close();
+      hcv.close();
+    }
+  }
+
+  @Test
+  void testConcatLists() {
+    List<Integer> list = new ArrayList<>();
+    list.add(0);
+    list.add(1);
+    list.add(2);
+    list.add(3);
+    List<Integer> list2 = new ArrayList<>();
+    list2.add(6);
+    list2.add(2);
+    list2.add(4);
+    list2.add(5);
+    List<Integer> list3 = new ArrayList<>();
+    list3.add(0);
+    list3.add(7);
+    list3.add(3);
+    list3.add(4);
+    list3.add(2);
+
+    List<Integer> list4 = new ArrayList<>();
+    list4.add(10);
+    list4.add(11);
+    list4.add(12);
+    list4.add(13);
+    List<Integer> list5 = new ArrayList<>();
+    list5.add(16);
+    list5.add(12);
+    list5.add(14);
+    list5.add(15);
+    List<Integer> list6 = new ArrayList<>();
+    list6.add(100);
+    list6.add(107);
+    list6.add(103);
+    list6.add(104);
+    list6.add(200);
+    try (ColumnVector res1 = ColumnVector.fromLists(DType.INT32, list, list2, list3);
+         ColumnVector res2 = ColumnVector.fromLists(DType.INT32, list4, list5, list6);
+         ColumnVector v = ColumnVector.concatenate(res1, res2);
+         ColumnVector expected = ColumnVector.fromLists(DType.INT32, list, list2, list3, list4, list5, list6);
+         HostColumnVector hostColumnVector = v.copyToHost()) {
+      List<Integer> ret = hostColumnVector.getList(5);
+      System.out.println("Guess we passed!"+ret.size() + " v size ="+v.getRowCount());
+      //add asserts
+      assertColumnsAreEqual(expected,v);
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+
+  @Test
+  void testConcatListsStrings() {
+    List<String> list = new ArrayList<>();
```

Comment:
There are much cleaner ways to create a list.

```
static import java.utils.Arrays.asList;

...
try ( ColumnVector res1 = ColumnVector.fromLists(DType.STRING,
                                         asList("0", "1", "2", "3"),
                                         null);
        ColumnVector res2 = ColumnVector.fromLists(DType.STRING,
                                         asList("4", null, "6", null));
...
```

Reply:
Agree. Will get this in the next round.